### PR TITLE
Editing - Run Kibana on ECK

### PR DESCRIPTION
**Pull request by alaudazzi**
This PR suggests the following editing:

- Eliminate one redundant topic in the navigation tree ([Connect to an Elasticsearch cluster](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-kibana-connect-to-es.html))
- Harmonize capitalization across titles
- Simplify some convoluted sentences

## Question for reviewers:

Can you help clarify/rework this sentence:

> If this behavior is not desired, more than one ECK instance can be deployed.

Relates to [#65635](https://github.com/elastic/cloud/issues/65635)

### CODE REVIEW

File: docs/orchestrating-elastic-stack-applications/kibana.asciidoc [hunk 1]
```diff
@@ -224,14 +220,13 @@ spec:
...
-== HTTP Configuration
+== HTTP configuration
 
 [id="{p}-kibana-http-publish"]
 === Load balancer settings and TLS SANs
 
 By default a `ClusterIP` link:https://kubernetes.io/docs/concepts/services-networking/service/[service] is created and associated with the Kibana deployment.
-You may want to expose Kibana externally with a link:https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer[load balancer].
-In which case you may also want to include a custom DNS name or IP in the self-generated certificate.
+If you want to expose Kibana externally with a link:https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer[load balancer], you have to include a custom DNS name or IP in the self-generated certificate.
```
Comment (pebrc) [inline docs/orchestrating-elastic-stack-applications/kibana.asciidoc:229]: > you have to include a custom DNS name or IP in the self-generated certificate.

You don't have to. But if you don't you will see certificate errors when accessing Kibana in a browser. I think that's why it was worded a little vague in the original. I would still weaken the statement a bit: maybe: "it is recommended to set a custom DNS name or IP"?

**Comment by alaudazzi** [comment]
Thank you for your feedback @pebrc. Can you also help with this sentence:

> If this behavior is not desired, more than one ECK instance can be deployed.

It's a bit weird, I would reword it but I don't understand it.

**Comment by pebrc** [comment]
> Thank you for your feedback @pebrc. Can you also help with this sentence:
> 
> > If this behavior is not desired, more than one ECK instance can be deployed.
> 
> It's a bit weird, I would reword it but I don't understand it.

What the author was trying to say here is that if you want perfect isolation and make it impossible for a Kibana from another namespace to connect to your Elasticsearch deployments you can achieve that by installing multiple instances of ECK, one for your namespace and then another instance for that other namespace where the potentially rogue Kibana is located. This "fixes" the problem because the association mechanism works only within the area of responsibility of a single ECK instance. 

But I believe this statement is actually outdated and we have since introduced a mechanism to restrict cross-namespace relationships which is documented here: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-restrict-cross-namespace-associations.html

File: docs/orchestrating-elastic-stack-applications/kibana.asciidoc [hunk 2]
```diff
@@ -46,16 +42,16 @@ spec:
...
 
-NOTE: `namespace` is optional if the Elasticsearch cluster is running in the same namespace as Kibana.
+The use of `namespace` is optional if the Elasticsearch cluster is running in the same namespace as Kibana.
 
-NOTE: Any Kibana can reference (and thus access) any Elasticsearch instance as long as they both are in namespaces that are watched by the same ECK instance. ECK will copy the required `Secret` from Elasticsearch to Kibana namespace. If this behavior is not desired, more than one ECK instance can be deployed. Kibana won't be able to automatically connect to Elasticsearch (through `elasticsearchRef`) in a namespace managed by a different ECK instance.
+NOTE: Any Kibana can reference (and thus access) any Elasticsearch instance as long as they are both in namespaces that are watched by the same ECK instance. ECK will copy the required Secret from Elasticsearch to Kibana namespace. Kibana cannot automatically connect to Elasticsearch (through `elasticsearchRef`) in a namespace managed by a different ECK instance. For more information about cross-namespace relationship, see <<{p}-restrict-cross-namespace-associations,Restrict cross-namespace resource associations>>.
```
[the remark this answers is not in the document]
Comment (pebrc) [reply docs/orchestrating-elastic-stack-applications/kibana.asciidoc]: I am sorry, but my suggestion is flawed.  I just realised that we now have "restrict cross-namespace associations" twice, once in the sentence and once in the link. Do you think it is worth removing the duplication?

[the remark this answers is not in the document]
Comment (alaudazzi) [reply docs/orchestrating-elastic-stack-applications/kibana.asciidoc]: Duplication removed.

**Comment by pebrc** [comment]
Almost good to merge 👍 , just one more thing: I just rendered the asciidoc and noticed that it still says "with ECK" and "without ECK" in the headings. Also I am not sure if the removal of the "Connect to an Elasticsearch cluster" entry in the navigation tree is an improvement

Screenshot 2020-09-04 at 09 26 34


I think the current version (below) looks less cluttered in the navigation tree and avoids the long headings in the navigation. 

Screenshot 2020-09-04 at 09 29 45

**Comment by alaudazzi** [comment]
This is why I initially replace "managed by" with "with ECK". I can add some introductory text between H2 and H3 to make the text flow more natural.

**Comment by alaudazzi** [comment]
I personally find the sequence of titles and text terribly redundant, but we can live with that for now.

**Review by pebrc — APPROVED** [review]
Agreed there is some redundancy. I wonder if we could vary the headings: 

* Connect to an Elasticsearch cluster
  * Elasticsearch is managed by ECK
  * Elasticsearch is not managed by ECK

Or something along those lines.
